docs/sources/developer/architecture/components/metas.md

```diff
@@ -0,0 +1,107 @@
+# Metas
+
+Metas are key-value properties that are sent with each payload.
+
+Metas can be either:
+
+- static - once defined, they are not changing across a session
+- dynamic - either computed automatically for each payload or they are changed programmatically by the end-user.
+
+## Available Metas
+
+### App
+
+The `app` meta ties signals with a specific app and it should not be changed across a session. It is required during
```

Comment:
It means that this is not modified across the same session but it can change between sessions. This should not be modified by neither the user nor it is changing automatically.